examples/tensorflow/object_detection/README.md

```diff
@@ -160,20 +160,4 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 ## Results
 <a name="results"></a>
 
-|Model|Compression algorithm|Dataset|mAP (_drop_) %|NNCF config file|Checkpoint|
```

Comment:
Please adjust the link in the "Run Object Detection Sample" paragraph. I would recommend adding direct links to pre-trained weights for RetinaNet and YOLO v4 in this README.
```
 Download the pre-trained weights in H5 format and provide the path to them using `--weights` flag. The link to the
archive with pre-trained weights can be found in the `TensorFlow checkpoint` column of the **[results](#results)** table.
```

Reply:
Done